diem-move/diem-framework/experimental/releases/artifacts/current/build/DiemExperimental/sources/Roles.move

```diff
@@ -226,8 +230,11 @@ module DiemFramework::Roles {
     }
 
     public fun get_role_id(a: address): u64 acquires RoleId {
-        assert!(exists<RoleId>(a), Errors::not_published(EROLE_ID));
-        borrow_global<RoleId>(a).role_id
+        if (exists<RoleId>(a)) {
+            borrow_global<RoleId>(a).role_id
```

Comment:
Honestly, I'd prefer if we don't use single letter variables.  They don't read well & make it harder to understand.